crates/l2/contracts/src/l1/CommonBridge.sol

```diff
@@ -49,6 +50,18 @@ contract CommonBridge is
     /// @dev It's used to validate withdrawals
     address public constant L2_BRIDGE_ADDRESS = address(0xffff);
 
+    /// @notice How much of each L1 token was deposited to each L2 token.
+    /// @dev Stored as L1 -> L2 -> amount
+    /// @dev Prevents L2 tokens from faking their L1 address and stealing tokens
+    mapping (address => mapping (address => uint256)) public depositsERC20;
+
+    /// @notice Function of the L2 bridge to call for minting ERC20 tokens
+    /// @dev Selector of the mintERC20 function in the CommonBridgeL2 contract
+    bytes4 public constant L2_MINTERC20_SELECTOR = 0x79204fe0;
```

Comment:
Is there any chance we can use `CommonBridgeL2.mintERC20.selector` for this? I want to avoid errors due to forgetting to update this constant.